airbyte-cdk/python/setup.py

```diff
@@ -15,7 +15,7 @@
 
 setup(
     name="airbyte-cdk",
-    version="0.29.3",
+    version="0.29.1",
```

Comment:
0.29.1 is the last released version https://pypi.org/manage/project/airbyte-cdk/release/0.29.1/